examples/config_files/settings/customizing_settings.rst

```diff
@@ -0,0 +1,146 @@
+.. _examples_config_files_customizing_settings:
+
+Customizing settings
```

Comment:
I am missing the ``settings_user.yml`` new 2.0 feature, at least a place holder to fill it later.